### DISCUSSION TITLE
have option to show results screen on fail

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13093] [Ideas]
i suggest that when the option is set to true, if the player fails a map, instead of just the failed screen appearing, it will transition to the results screen. The results screen in the failed case will give you an F instead of any other grade(because you failed) and it will show the % of the map completed. The results screen will be normal otherwise.

**Comment by peppy** [comment]
Probably won't be the results screen but some additional information on the fail overlay.
